dashboard/db/migrate/20170127183604_create_sign_ins.rb

```diff
@@ -0,0 +1,13 @@
+class CreateSignIns < ActiveRecord::Migration[5.0]
+  def change
+    create_table :sign_ins do |t|
```

Comment:
The powers that be agree that we do not want to store IP addresses in this table.